docs/source/user/fast.farm/InputFiles.rst

```diff
@@ -408,6 +408,7 @@ The wake will adopt a "curled" shape in skewed inflow.
 When Wake_Mod=2,3, the stability of the algorithm will depend on the choice of **dr** and **DT_Low** (see the guidelines (see the guidelines given in :numref:`FF:ModGuidance`).
 
 
+**RotorDiamRef** [float, in m]: Reference turbine rotor diameter for wake calculations. 
```

Comment:
How is RotorDiamRef used in the code?  Does it replace EstimatedRotorRadius and/or D_rotor_filt?

Reply:
I've added the following: 

It is used to:
verify that the high-res boxes are sufficiently large cover the rotor, 
determine the default value of **f_c** (see below), 
determine the physical grid spacing of the `WAT_BoxFile` when **WAT=1** (see below).